### PR TITLE
Create Welsh version for check or update your company car tax

### PR DESCRIPTION
**Pull request by ben-strange**
https://trello.com/c/SsXmyXS0/1086-update-company-car-details-welsh-content-version-content-change-request

### CODE REVIEW

File: lib/service_sign_in/check-update-company-car-tax.cy.yaml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    [Creu cyfrif Porth y llywodraeth.]((https://www.tax.service.gov.uk/paye/company-car/start-government-gateway])
+
+    ###GOV.UK Verify
+
+    Fel arfer mae'n cymryd tua 15 munud i gofrestru gyda GOV.UK Verify. Bydd yn gweithio orau os oes gennych y canlynol:
+
+    - cyfeiriad yn y DU
+    - pasbort neu drwydded cerdyn-llun dilys
+
+    [Create a GOV.UK Verify account](https://www.signing.service.gov.uk/dechrau )
```
[the remark this answers is not in the document]
Comment (ben-strange) [reply lib/service_sign_in/check-update-company-car-tax.cy.yaml]: Thanks for spotting these things, Jakub. I'll make the changes.

[the remark this answers is not in the document]
Comment (ben-strange) [reply lib/service_sign_in/check-update-company-car-tax.cy.yaml]: Have made the changes if you'd be able to review for me? Thanks.

File: lib/service_sign_in/check-update-company-car-tax.cy.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    ##Dewiswch ffordd o brofi pwy ydych
+
+    ###Porth y Llywodraeth
+
+    Fel arfer mae'n cymryd tua 10 munud i greu cyfrif Porth y Llywodraeth. Bydd yn gweithio orau os oes gennych y canlynol:
+
+    - eich rhif Yswiriant Gwladol
+    - slip cyflog diweddar, P60 neu basbort y DU sy'n ddilys
+
+    [Creu cyfrif Porth y llywodraeth.]((https://www.tax.service.gov.uk/paye/company-car/start-government-gateway])
```
Comment (jakubmiarka) [inline lib/service_sign_in/check-update-company-car-tax.cy.yaml]: `llywodraeth.]((https://` double opening brackets there

File: lib/service_sign_in/check-update-company-car-tax.cy.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+locale: cy
+choose_sign_in:
+  title: Profi pwy ydych er mwyn mynd yn eich blaen
+  slug: profi-pwy-ydych
+  options:
+    - text: Defnyddio Porth y Llywodraeth
+      url: https://www.tax.service.gov.uk/account
+      hint_text: Bydd gennych Ddynodydd Defnyddiwr (ID) os ydych wedi cofrestru i wneud pethau fel cyflwyno'ch Ffurflen Dreth Hunanasesiad ar-lein.
+    - text: Defnyddio GOV.UK Verify
+      url: https://www.tax.service.gov.uk/ida/sa/login
```
Comment (jakubmiarka) [inline lib/service_sign_in/check-update-company-car-tax.cy.yaml]: Same thing as above, I would go with `https://www.tax.service.gov.uk/paye/company-car/start-verify` to stay consistent and make sure if the internal address changes, HMRC can still handle/redirect our traffic

**Comment by jakubmiarka** [comment]
@ben-strange Hi Ben, I've added three more comments. Other than that I think it should be fine. I work on Verify so I'm not authorised to approve.

**Comment by leenagupte** [comment]
@ben-strange We can merge this when your changes are ready to be published.

**Pull request merged**